config.go

```diff
@@ -23,7 +23,7 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2018-04-10"
+	ClientVersion = "2018-04-25"
```

Comment:
given that the feature isn't actually usable at this point, I might hold off bumping the cli version, as users will see no benefit if we force update.